src/commands_controller.ts

```diff
@@ -108,7 +109,9 @@ export class CommandsController implements Disposable, NeovimExtensionRequestPro
     };
 
     private scrollLine = (to: "up" | "down"): void => {
-        vscode.commands.executeCommand("editorScroll", { to, by: "line", revealCursor: false });
+        const settings = vscode.workspace.getConfiguration(EXT_NAME);
```

Comment:
Pass the option as constructor param from the main_controller, don't query it every time for perf reasons